card/lib/card/format/nest/subformat.rb

```diff
@@ -27,6 +27,11 @@ def field_subformat field
           field = card.name.field(field) unless field.is_a?(Card)
           subformat field
         end
+
+        def inherit variable
```

Comment:
this new api probably warrants a comment.  it means take the instance variable from me or the first ancestor that has it.